content/en/docs/getting-started/_index.md

```diff
@@ -1,20 +1,54 @@
 ---
 title: Getting Started
 description: Getting started with Falco
+hide_section_index: true
 weight: 10
 ---
  
-Falco is a cloud-native security tool. It provides near real-time threat detection for cloud, container, and Kubernetes workloads by leveraging runtime insights. Falco can monitor events from various sources, including the Linux kernel, and enrich them with metadata from the Kubernetes API server, container runtime, and more.
+Falco is a cloud-native security tool. It provides near real-time threat detection for cloud, container, and Kubernetes workloads by leveraging runtime insights. Falco can monitor events defined via customizable {{< glossary_tooltip text="rules" term_id="rules" >}} from various sources, including the Linux kernel, and enrich them with metadata from the Kubernetes API server, container runtime, and more. Falco supports a wide range of kernel versions, x86_64 and ARM64 architectures, and many different {{< glossary_tooltip text="output" term_id="outputs" >}} channels.
```

Comment:
Good catch. Moreover, the `outputs` definition in the glossary is misleading (since the glossary definition just refers to the `output:` key in the rule, not just about outputs in general).

I will fix it here by removing the tooltip for now. Then, I guess we need a follow-up PR to address this inconsistency in the glossary and whenever the `outputs` tooltip is used.

Reply:
cc @LucaGuerra